src/gbitmap/gbitmap.c

```diff
@@ -0,0 +1,201 @@
+#include "gbitmap.h"
+#include "macros.h"
+
+static const uint8_t n_prv_palette_size[] = {
+    0, /* n_GBitmapFormat1Bit */
```

Comment:
can use `[1] = ...` gcc extension syntax to avoid enum order dependency, I think

Reply:
Cool, I didn't know this syntax yet!